kubejs/server_scripts/enigmatica/kubejs/constants/materials.js

```diff
@@ -173,15 +173,15 @@ const oreProcessingSecondaries = {
         createProcessingTime: 400
     },
     cloggrum: {
-        secondary: 'lead',
+        secondary: 'froststeel',
```

Comment:
I just changed those to match how Materialis did them since it already had them supported, same thing with the temperature changes. Want me to change them back?

Reply:
Ah I see, cool. I'm not sure tbh. @MuteTiefling what do you think?